daemon/control.proto

```diff
@@ -246,7 +246,9 @@ message DaemonToController {
 
 		CONTAINER_CMLD_HANDLES_PIN = 7; // -> [container_cmld_handles_pin]
 
-		LOG_MESSAGE = 12;		// -> [log_message]
+		LOG_MESSAGE_FRAGMENT = 12;		// -> [log_message]
```

Comment:
Lets reuse one of the legacy codes of never implemented logging feature.
LOG_MESSAGE_FRAGMENT = 11, LOG_MESSAGE_FINAL = 12
see reserved